tutorials/algorithms/02_vqe_advanced_options.ipynb

```diff
@@ -0,0 +1,572 @@
+    "\n",
+    "One might wonder... *Why set yourself an initial point?* Well, this option can come in handy if you have a guess for a reasonable starting point for the problem or perhaps know information from a prior experiment.\n",
+    "\n",
+    "To demonstrate this feature, let's look at the results from our previous VQE run:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "metadata": {
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "{   'aux_operators_evaluated': None,\n",
+      "    'cost_function_evals': 9,\n",
+      "    'eigenvalue': -1.8572750175655812,\n",
+      "    'optimal_circuit': <qiskit.circuit.library.n_local.two_local.TwoLocal object at 0x13cab3280>,\n",
+      "    'optimal_parameters': {   ParameterVectorElement(θ[3]): 6.092947832767056,\n",
+      "                              ParameterVectorElement(θ[4]): -2.598326651673288,\n",
+      "                              ParameterVectorElement(θ[2]): 0.5470777607659094,\n",
+      "                              ParameterVectorElement(θ[7]): 0.36021017470898664,\n",
+      "                              ParameterVectorElement(θ[6]): -4.717616147449751,\n",
+      "                              ParameterVectorElement(θ[5]): 1.5683250498282322,\n",
+      "                              ParameterVectorElement(θ[1]): 4.426962358395531,\n",
+      "                              ParameterVectorElement(θ[0]): 4.296519450348719},\n",
+      "    'optimal_point': array([ 4.29651945,  4.42696236,  0.54707776,  6.09294783, -2.59832665,\n",
+      "        1.56832505, -4.71761615,  0.36021017]),\n",
+      "    'optimal_value': -1.8572750175655812,\n",
+      "    'optimizer_evals': None,\n",
+      "    'optimizer_result': <qiskit.algorithms.optimizers.optimizer.OptimizerResult object at 0x13cab29e0>,\n",
+      "    'optimizer_time': 0.35492467880249023}\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(result)\n",
+    "optimizer_time = result.optimizer_time"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now, we can take the `optimal_point` from the above result and use it as the `initial_point` for a follow-up computation.\n",
+    "\n",
+    "**Note:** `initial_point` is now a keyword-only argument of the `VQE` class (i.e, it must be set following the `keyword=value` syntax)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "metadata": {
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "{   'aux_operators_evaluated': None,\n",
+      "    'cost_function_evals': 9,\n",
```

Comment:
In the old tutorial the runs were more equivalent. If you look the one took 65 evals with SLSQP and the one at the optimal point just 9. Both here seem to take just 9. I think thats because before the internal optimizers finite diff was used which computes the gradient with the objective function - which is the latter case here too when the optimal point is given. Which explains the difference in time despite the number or evals in this case being the same. Maybe it would be better to run this latter one using the gradient to be more directly comparable with the reference result. I imagine then it would be just be 1 eval since its at the optimal point the gradient will cause it to to terminate since its already at the minumum.

Reply:
Good point, I just assumed the new gradients modified the number of evals.